app/Http/Requests/LoginRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Services\ResponseService;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Http\Exceptions\HttpResponseException;
+
+class LoginRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'email' => 'required|email',
+            'password' => 'required'
+        ];
+    }
+
+    protected function failedValidation(\Illuminate\Contracts\Validation\Validator $validator)
```

Comment:
Instead of having this method duplicated across all the request form, you could overwrite laravel's "transformErrors" method in your Exception/Handler.php file. It could look something like this. 

`/**
     * Re-format validation error to return in apporitate format
     *
     * @param \Illuminate\Validation\ValidationException  $exception
     * @return array $errors
     */
    private function transformErrors(ValidationException $exception)
    {
        $errors = [];
        foreach ($exception->errors() as $field => $message) {

         //You can adjust this array just anyhow you'll want your response to look like.
            $errors[] = [
               'field' => $field,
               'error_message' => $message[0],
            ];
        }
        return $errors;
    }`

Reply:
Done, thanks.